src/server/api/mutations/sendMessage.js

```diff
@@ -139,17 +140,14 @@ export const sendMessage = async (
   }
   contact.message_status = saveResult.contactStatus;
 
-  // log.info(
-  //   `Sending (${serviceName}): ${messageInstance.user_number} -> ${messageInstance.contact_number}\nMessage: ${messageInstance.text}`
-  // );
-  //NO AWAIT: pro=return before api completes, con=context needs to stay alive
-  service.sendMessage(
-    saveResult.message,
+  await jobRunner.dispatchTask(Tasks.SEND_MESSAGE, {
+    message: saveResult.message,
     contact,
-    null,
+    // TODO: trx can't be supported with remote dispatch, see if it's necessary
```

Comment:
I don't think trx is necessary when called from here, anyway.  I wonder if instead of all our 'if trx....` from twilio.sendMessage, we should just start the transaction *inside* sendMessage (not in this PR, but later)